Documentation/parameters.par

```diff
@@ -15,10 +15,16 @@ cICType = 'Zero'                  ! Initial Condition
 !Possible values are
 !Zero        All zero - the standard case to work with point source or dynamic rupture
 !Planarwave  A planar wave for convergence tests, needs periodic boundary conditions
+!Travelling  A planar wave travelling wave (one period of a plane wave), needs more parameters to be specified
 !Scholte     A Scholte wave to test elastic-acoustic coupling
 !Snell       Snells law to test elastic-acoustic coupling
 !Ocean       An uncoupled ocean test case for acoustic equations
 
+!The following parameters are only needed for the travelling wave IC:
+origin = 0 0 0.5             ! Origin of the wave 
+kVec = 6.283 0 0             ! Gives direction of wave propagation, the wavelength of the travelling wave is 2*pi / norm(kVec)
+ampField = 2 0 0 0 0 0 0 1 0 ! Amplification of the different wave mod
```

Comment:
I don't understand this. What does the amp. field mean exactly? It probably states which variables you want to init, right?

Reply:
Compare to gamma in Eq. 35 in https://library.seg.org/doi/10.1190/1.2965027, but I'll explain better ;-)